### PR TITLE
also set intra_op_parallelism_threads

### PR DESCRIPTION
**Pull request by zachmayer**
### Summary
Conveniently, keras uses`os.environ.get('OMP_NUM_THREADS')` to set `intra_op_parallelism_threads`.  I think it would be useful to also set `inter_op_parallelism_threads` using this variable.

This is convenient because e.g. you can set `OMP_NUM_THREADS=1` and keras will use no multithreading.

### PR Overview

- [x] This PR requires new unit tests [y/n] (make sure tests are included)
- [ ] This PR requires to update the documentation [y/n] (make sure the docs are up-to-date)
- [x] This PR is backwards compatible [y/n]
- [x] This PR changes the current API [y/n] (all API changes need to be approved by fchollet)

### CODE REVIEW

**Comment by zachmayer** [comment]
I'm open to feedback on the test.  I can either do `@pytest.mark.skipif(K.backend() != 'tensorflow')` or use `KTF`.

File: keras/backend/tensorflow_backend.py [hunk 1]
```diff
@@ -186,6 +186,7 @@ def get_session():
             else:
                 num_thread = int(os.environ.get('OMP_NUM_THREADS'))
                 config = tf.ConfigProto(intra_op_parallelism_threads=num_thread,
+                                        inter_op_parallelism_threads=num_thread,
```
Comment (fchollet) [inline keras/backend/tensorflow_backend.py]: What is this meant to achieve? Is this always the right behavior?

Comment (zachmayer) [reply keras/backend/tensorflow_backend.py]: This is meant to achieve reproducible keras models, with the tensorflow backend, if the environment variable `OMP_NUM_THREADS` is set to 1.

In other words, if the user sets `OMP_NUM_THREADS=1` the tensorflow backend will use no multi threading and will make models that are reproducible.  (Using multiple threads with the tensorflow backend can create issues with reproducibility, even if the random seed is set).

If you'd prefer, I can add a new environment variable, e.g. `TF_INTER_OP_PARALLELISM_THREADS`.

My goal is to be able to set both `intra_op_parallelism_threads` and `inter_op_parallelism_threads` from an environment variable, so I can set them without having to manually manage tensorflow sessions.

Does that make sense?

Comment (zachmayer) [reply keras/backend/tensorflow_backend.py]: @fchollet What do you think?  I could also make a new environment variable, for setting `TF_INTER_OP_PARALLELISM_THREADS`.

**Comment by zachmayer** [comment]
Awesome, thank you!
